docs/x509/verification.rst

```diff
@@ -297,3 +302,167 @@ the root of trust:
             for server verification.
 
         :returns: An instance of :class:`ClientVerifier`
+
+.. class:: ExtensionPolicy
+
+    .. versionadded:: 44.0.0
+
+    An ExtensionPolicy provides a builder-style interface for constructing an 
+    :class:`ExtensionPolicy`.
+
+    .. staticmethod:: permit_all()
+
+        Creates an ExtensionPolicy initialized with a policy that does 
+        not put any constraints on a certificate's extensions. 
```

Comment:
I'd maybe add a sentence that expresses that this is what you call if you're trying to do something custom.